tagger/train/train.py

```diff
@@ -28,14 +28,17 @@
 )
 
 # GLOBAL PARAMETERS TO BE DEFINED WHEN TRAINING
-tf.keras.utils.set_random_seed(420) #not a special number 
+tf.keras.utils.set_random_seed(420) # not a special number 
 BATCH_SIZE = 1024
-EPOCHS = 100
-VALIDATION_SPLIT = 0.1 # 10% of training set will be used for validation set. 
+EPOCHS = 200
+VALIDATION_SPLIT = 0.2 # 20% of training set will be used for validation set. 
+LOSS_WEIGHTS = [1., 1.]
+WEIGHT_METHOD = "onlyclass"
+DEBUG = True
```

Comment:
I think this should be a command line variable rather than a hardcoded constant and added to the argument parser